<launch>

  <arg name="rik" default="false"/>
  <arg name="lik" default="true"/>

  <arg name="info_file" default="ur3e_info.yaml"/>

  <!-- Custom test code -->
<!--
  <node pkg="robotiq_85_driver" type="robotiq_85_test" name="robotiq_85_test" output="screen">
      <param name="num_grippers" value="1"/>
  </node>
-->
<!--
  <node pkg="driver_test" type="direct_joint_node.py" name="direct_joint_node" output="screen">
    <param name="info_file_path" value="$(find relaxed_ik)/src/RelaxedIK/Config/info_files/$(arg info_file)"/>
  </node>
-->

  <node if="$(arg rik)" pkg="ur3e_official_driver_relaxed_ik_demo" type="driver_test_node_rik.py" name="driver_test_node" output="screen">
    <param name="info_file_path" value="$(find relaxed_ik)/src/RelaxedIK/Config/info_files/$(arg info_file)"/>
  </node>

  <node if="$(arg lik)" pkg="ur3e_official_driver_relaxed_ik_demo" type="driver_test_node_lik.py" name="driver_test_node" output="screen">
    <param name="info_file_path" value="$(find lively_ik)/src/RelaxedIK/Config/info_files/$(arg info_file)"/>
  </node>

  <node pkg="ur3e_official_driver_relaxed_ik_demo" type="gripper_buttons.py" name="gripper_buttons" output="screen"/>


  <!-- Rviz -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ur3e_official_driver_relaxed_ik_demo)/rviz/test.rviz" />

</launch>
